src/client/renderer/r_types.h

```diff
@@ -359,7 +359,7 @@ typedef struct r_material_s {
 	// from media
 	r_media_t media;
 
-	const struct cm_material_s *cm; // link to cm material
+	struct cm_material_s *cm; // copy of cm material
```

Comment:
It was a copy initially and then I just realized I can reference the data like I did in the Bsp subsystem, so I did that instead.